content/reference/current/user/binary/index.md

```diff
@@ -169,19 +169,51 @@ An artifact will be published will the `tests` classifier.
 
 ### SBT
 
+#### With the `test` configuration
+
+Add the following to your `build.sbt` configuration:
+
 ```scala
-packageBin in Test := (assembly in Test).value
-publishArtifact in Gatling := true
-publishTo :=
-	(if (isSnapshot.value)
-		Some("private repo" at "REPLACE_WITH_YOUR_SNAPSHOTS_REPOSITORY_URL")
-	else
-		Some("private repo" at "REPLACE_WITH_YOUR_RELEASES_REPOSITORY_URL")
+Test / packageBin := (Gatling / enterprisePackage).value
+Test / publishArtifact := true
+Compile / publishArtifact := false // If you only want to publish Gatling tests from this project (nothing from src/main)
+publishTo := (
+  if (isSnapshot.value)
+    Some("private repo" at "REPLACE_WITH_YOUR_SNAPSHOTS_REPOSITORY_URL")
+  else
+    Some("private repo" at "REPLACE_WITH_YOUR_RELEASES_REPOSITORY_URL")
 )
 ```
 
+You can deploy the test jar with the following command:
+
 ```shell
-sbt test:publish
+sbt Gatling/publish
```

Comment:
In reality, `Gatling/publish` just delegates to `Test/publish`, which itself just delegates to `publish` - it's not actually scoped to anything except the project.

I think we can just document the command `sbt publish`, since at least it's clear that it publishes the entire project. I don't think it usually makes much sense to publish only part of a project anyway?